packages/ecc-utils-design/package.json

```diff
@@ -1,8 +1,6 @@
 {
   "name": "@elixir-cloud/design",
   "description": "Abstraction layer that provides utility components to manage the headless yet constitent design across ecc component",
-  "license": "Apache-2.0",
```

Comment:
please change the license, version, repository, files back to how it is in the cleanup branch. I assume its changes because you might have worked on previous commit.